core/src/crds_gossip_pull.rs

```diff
@@ -408,7 +437,9 @@ impl CrdsGossipPull {
         for (value_hash, _insert_timestamp) in &self.purged_values {
             add_value_hash(&value_hash);
         }
-
+        for (value_hash, _) in &self.failed_inserts {
```

Comment:
Hmm, since the filter is changed, does it need any kind of gating for mixed-network nodes?